R/utils-fetch.R

```diff
@@ -126,12 +126,9 @@ fetchCorrections <- function(reportObject, seriesCorrName){
 #' @param reportObject the full report data
 #' @param stat 'MIN' or 'MAX' data to select
 fetchMinMaxIVs <- function(reportObject, stat){
-  if(!is.null(stat)){
```

Comment:
I removed it because the other fetch functions don't do NULL checking either. The NULL checking is done from the read function. Might be worth discussing with everyone if fetch functions should do NULL checking or not though.